Sources/Subprocess/IO/Input.swift

```diff
@@ -168,6 +168,15 @@ extension InputProtocol where Self == FileDescriptorInput {
             closeAfterSpawningProcess: closeAfterSpawningProcess
         )
     }
+
+    /// Create a Subprocess input that reads from the standard input of
+    /// current process. The file descriptor should not be closed afterwards.
```

Comment:
Should this say "will not be closed" rather than "should not be closed"?

Reply:
Preference for not using the future tense:
```suggestion
    /// current process. The file descriptor isn't closed afterwards.
```

Reply:
Thanks @heckj !!